lang/cs/Org.Apache.REEF.Client/Org.Apache.REEF.Client.csproj

```diff
@@ -88,7 +98,7 @@ under the License.
     <Compile Include="YARN\RestClient\DataModel\ClusterInfo.cs" />
     <Compile Include="YARN\RestClient\DataModel\ClusterMetrics.cs" />
     <Compile Include="YARN\RestClient\DataModel\Error.cs" />
-    <Compile Include="YARN\RestClient\DataModel\MaximumResourceCapability.cs" />
+    <Compile Include="YARN\RestClient\DataModel\Resouce.cs" />
```

Comment:
Is it auto generated? If you would like to use another name, can you change it in schema?